packages/fxa-auth-server/lib/senders/emails/storybook-email.ts

```diff
@@ -0,0 +1,51 @@
+export interface StorybookEmailArgs {
+  template: string;
+  description?: string;
+  variables: Record<string, any>;
+}
+
+export const storybookEmail = ({
+  template,
+  description,
+  variables,
+}: StorybookEmailArgs): HTMLDivElement => {
+  const container = document.createElement('div');
+  container.innerHTML = 'Loading email...';
+  const emailDescriptionDiv = document.createElement('div');
+  if (description) {
+    emailDescriptionDiv.innerHTML = `
+        <p>${description}</p>
+        <hr />`;
+  }
+
+  renderUsingMJML({ template, variables })
+    .then((result) => {
+      container.innerHTML = emailDescriptionDiv.innerHTML + result;
+    })
+    .catch((error: Error) => {
+      container.innerHTML = `Error loading email: ${error.message}`;
+    });
+
+  return container;
+};
+
+async function renderUsingMJML({
+  template,
+  apiUrl = 'http://localhost:8192',
+  variables,
+}: {
+  template: string;
+  apiUrl?: string;
+  variables: Record<string, any>;
+}): Promise<string> {
+  const resp = await fetch(apiUrl, {
+    method: 'POST',
+    body: JSON.stringify({
+      template,
+      ...variables,
+    }),
+  });
+  return await resp.text();
```

Comment:
Currently, if an error occurs on the rendering/server side the fetch operation will still complete, but the returned body will be an error message. In order to use the try/catch for this scenario in the `storybookEmail` function you could replace this line with:

```js
const result = await resp.text();
if (resp.status !== 200) {
  throw new Error(result);
}
return result;
```